src/main/java/edu/utdallas/davisbase/representation/CreateIndexCommandRepresentation.java

```diff
@@ -0,0 +1,58 @@
+package edu.utdallas.davisbase.representation;
+
+public class CreateIndexCommandRepresentation implements CommandRepresentation {
+  private String command;
+  private String table;
+  private String index;
+  private String column;
+
+  public CreateIndexCommandRepresentation(String command, String table, String index, String column) {
+    this.command = command;
+    this.table = table;
+    this.index = index;
+    this.column = column;
+  }
+
+  public String getTable() {
+    return table;
+  }
+
+  public void setTable(String table) {
```

Comment:
Remove the setters to make this object immutable. _(it provides stronger compile-time correctness guarantees, and we don't the setters after construction anyway)_